dissect/target/plugin.py

```diff
@@ -218,6 +218,34 @@ def __init_subclass__(cls, **kwargs):
     def __init__(self, target: Target):
         self.target = target
 
+    def is_compatible(self) -> bool:
+        """Perform a compatibility check with the target.
+
+        Perform a compatibility check with the target
+        without raising any exceptions and optionally in a
+        more perrformant manner
+
+        """
```

Comment:
```suggestion
        """Perform a compatibility check with the target."""
```
I personally don't find that the rest of the text adds useful information.